### ISSUE TITLE
Specifying server-side encryption and Reduced Redundancy storage

### ISSUE THREAD

**Issue by ykessler** [waynehoover/s3_direct_upload#138]
Is it possible to include options for specifying [server-side encryption](http://docs.aws.amazon.com/AmazonS3/latest/dev/SSEUsingRESTAPI.html) and Reduced Redundancy Storage (RRS)?

**Comment by mogox** [comment]
Added an example on the wiki page based on this conversation: https://github.com/waynehoover/s3_direct_upload/wiki/Example-of-setting-up-%22s3_direct_upload%22-with-Server-Side-Encryption-on-Amazon-S3

However, I still think @ilyakatz PR #225 makes sense.

**Comment by elsurudo** [comment]
@mogox Agreed – it would be nice to be able to specify a single option, perhaps even on an initializer or something. This method works, but it's a bit verbose.
